src/custom/utils/operator/index.ts

```diff
@@ -191,31 +188,54 @@ export async function getPriceQuote(params: PriceQuoteParams): Promise<PriceInfo
   const [checkedBaseToken, checkedQuoteToken] = [checkIfEther(baseToken, chainId), checkIfEther(quoteToken, chainId)]
   console.log('[util:operator] Get Price from API', params)
 
-  return _getJson(
-    chainId,
-    `/markets/${toApiAddress(checkedBaseToken, chainId)}-${toApiAddress(checkedQuoteToken, chainId)}/${kind}/${amount}`,
-    'Quote'
-  )
+  try {
```

Comment:
@alfetopito is this better you think? handling the errors in the respective functions?

Reply:
Ahhh yes, feels good :feelsgood: